articles/File_download.md

```diff
@@ -0,0 +1,60 @@
+# Download file from a website and upload it to Dropbox 
+
+## My very first function in powershell ~
+
+```powershell
+function Get-UrlInfo {
+        [Cmdletbinding()]
+        param(
+                [Parameter(Mandatory=$true)]
+                [String]$fileName
+
+        )
+        $fileContent = Get-Content $fileName
+        $fileString = $fileContent.Split("</p>")[1] + "</p>"
+        $xmlHeader= [xml]$fileString
+        $hrefString = $xmlHeader.p.a.href
+        $key = ([System.Uri]$hrefString).Query.Split("=")[1]
+        $url = "https://fast.wistia.net/embed/iframe/" + $key
+
+        # Download the video from the above URL
+        youtube-dl $url
+
+        # Get the name of the downloaded file
+        $outputFile = Get-ChildItem *.bin -Name
+        $fileString = $outputFile.Split("-")[0]+"-$key" + ".mp4"
+
+        # Rename the downloaded file
+        $renameFile = mv $outputFile $fileString
+
+        # Upload the file on to Dropbox
+        rclone copy *.mp4 remote:/Videos/ -v
```

Comment:
Better to be more specific about the actual file you want to upload rather than a general `*.mp4`.

Reply:
You must have noticed that after renaming the file from `.bin` to `.mp4`, I am storing it in a variable **$renameFile**. However, I tried using this variable to copy the file, but it didn't work.
`rclone copy $renameFile remote:/Videos/ -v`
Error - Unable to get the file path

`rclone copy ./$renameFile remote:/Videos/ -v`
Worked - but uploaded all the files and folders from the current directory

Hence, I used this option. How would you do it?

Reply:
I'm not quite sure at the moment, would look into it later. As a reminder, I'm converting the above message into an `issue` as a reminder.